lib/archethic/shared_secrets/node_renewal_scheduler.ex

```diff
@@ -124,6 +124,9 @@ defmodule ArchEthic.SharedSecrets.NodeRenewalScheduler do
 
     ArchEthic.send_new_transaction(tx)
 
+    tx = NodeRenewal.new_origin_shared_secrets_transaction()
```

Comment:
Why do you want to create new origin shared secrets after each renewal of node shared keys ?
The node and origin shared secrets should be distinct I think

Reply:
This is just to temporary trigger for tx

Reply:
You can generate the transaction in the controller for now.

Reply:
OK